pyanaconda/modules/payloads/payload/payload_base.py

```diff
@@ -59,6 +63,29 @@ def type(self):
         """
         return None
 
+    @property
+    def side_payload(self):
+        """Get side payload attached to this payload.
+
+        The side payload is a payload attached to this payload. It can be configured by this
+        payload.
+
+        This side payload calls will be automatically queued for some DBus API in Payloads module.
```

Comment:
What does "some DBus API" mean in this context ?

Reply:
It depends on the specific API. It doesn't give sense to do this for each so it is documented for each method directly.